lua/cfc_notifications/client/cl_render.lua

```diff
@@ -59,14 +59,19 @@ hook.Add( "PlayerButtonDown", "CFCNotifications_render_keydown", function( ply,
     local x, _ = input.GetCursorPos()
     local wide = CFCNotifications.getSetting( "size_x" )
 
-    if key == KEY_F3 or ( clickerEnabled and key == MOUSE_LEFT and x < ( ScrW() - wide ) ) then
-        clickerEnabled = not clickerEnabled
-        local adjustCursor = CFCNotifications.getSetting( "adjust_cursor" )
-        if adjustCursor and clickerEnabled then
-            gui.SetMousePos( ScrW() - wide - 50, ScrH() / 2 )
+    if not DarkRP then
+        local shouldToggle = key == KEY_F3 or ( clickerEnabled and key == MOUSE_LEFT and x < ( ScrW() - wide ) )
```

Comment:
`shouldToggle = shouldToggle and not DarkRP`

?

Reply:
If only this was moon, finally a use for `and=`